test/GRAPHFILES

```diff
@@ -66,6 +66,7 @@ memleaks.graph
 studies/jacobi/jacobi.graph
 performance/elliot/no-op.graph
 performance/bharshbarg/forall-dom-range.graph
+performance/bharshbarg/arr-forall.graph
```

Comment:
This should also be listed under "# suite: Array/Domain performance" next to the other  performance/bharshbarg/forall-dom-range.graph